roles/openstack-setup/tasks/flavors.yml

```diff
@@ -21,6 +26,7 @@
       password: "{{ secrets.admin_password  }}"
   with_items: '{{ nova.flavors | default([]) }}'
   run_once: true
+  when: flavorcount.stdout == ""
   # FIXME flavors are always added, even if it's just an Ironic cloud
```

Comment:
Probably is best to see if we can iterate over nova.flavors (default flavor list in our envs) and for each see if flavor with that name exist. If it does not exist we create it.

Goal is to ensure we do not overwrite customer created flavor with same name as our default flavor and customer customized flavor's. If our default flavor does not exist we create it. Might have to use some ansible tircks to iterate over list, check if item.name is in `openstack flavor list -all -c Name -f value` and create item if not present.